### PR TITLE
week09_policy_II/ppo.ipynb

### PR DESCRIPTION
**Pull request by Alien-kz**
typo in loss function

### CODE REVIEW

**Comment by justheuristic** [comment]
image

File: week09_policy_II/ppo.ipynb [hunk 1]
```diff
@@ -472,7 +482,6 @@
     "\n",
     "* [Coefficient of Determination](https://en.wikipedia.org/wiki/Coefficient_of_determination) between \n",
     "value targets and value predictions\n",
-    "* Entropy of the policy $\\pi$\n",
```
Comment (dniku) [inline week09_policy_II/ppo.ipynb:475]: Why was this line removed?

Comment (Alien-kz) [reply week09_policy_II/ppo.ipynb:475]: In this task we do not use entropy component loss (it was used in week06, and think this paragraph is copy-paste typo).

**Comment by MichaelKonobeev** [comment]
Entropy would still be useful to monitor to understand how the training progresses even if it is not used in loss function definition.
